docs/modules/ROOT/pages/explanations/templating.adoc

```diff
@@ -0,0 +1,52 @@
+= Templating
+
+Some fields of the CRDs allow to use https://golang.org/pkg/text/template[Go templates] which are rendered by the operator.
+You can find some explanations on how to use them in here.
+
+
+== Cluster Template
+
+The tenant field `.spec.clusterTemplate` <<references/crds-html.adoc#syn.tools/v1alpha1.TenantSpec,defines>> a template which is used to set default values for clusters of this tenant.
+All _string_ fields within this allow to use Go templates which are rendered by the operator.
+As input data to the template rendering, the instance of the cluster on which the template is being applied on can be used.
+This data is a Go type defined https://github.com/projectsyn/lieutenant-operator/blob/34db9df5d2090a8840e69816d16ec13527c4fc2c/pkg/apis/syn/v1alpha1/cluster_types.go#L60[here].
+
+Since it's a Go type, only https://golang.org/ref/spec#Exported_identifiers[exported fields] (uppercase) can be accessed.
+
+[INFO]
```

Comment:
```suggestion
[NOTE]
```

INFO is not a known admonition. See https://asciidoctor.org/docs/user-manual/#admonition.